addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -30,6 +32,8 @@
 "200090","200090","Hire Purchase","liability_current","False"
 "200100","200100","Mortgages","liability_current","False"
 "210010","210010","Company Credit Card","liability_credit_card","False"
+"211008","211008","Creditors Suspense Account","asset_current","False"
+"212001","212001","Vat / Tax Control Account","liability_current","True"
```

Comment:
See 100060 & 200060
Should we remove those and replace with this one?

Reply:
I don't see the 100060 and 200060 accounts you're talking about

Reply:
it's on the current COA

Reply:
In my opinion it's better to keep the 100060 and 200060. I don't see any added value to replace them by the one suggested by the partner